client/CMT.cpp

```diff
@@ -350,24 +348,9 @@ int main(int argc, char * argv[])
 	session["aiSolo"].Bool() = false;
 	std::shared_ptr<CMainMenu> mmenu;
 	
-	if(vm.count("testmap"))
-	{
-		session["testmap"].String() = vm["testmap"].as<std::string>();
-		session["onlyai"].Bool() = true;
-		boost::thread(&CServerHandler::debugStartTest, CSH, session["testmap"].String(), false);
-	}
-	else if(vm.count("testsave"))
-	{
-		session["testsave"].String() = vm["testsave"].as<std::string>();
-		session["onlyai"].Bool() = true;
-		boost::thread(&CServerHandler::debugStartTest, CSH, session["testsave"].String(), true);
-	}

```

Comment:
why remove? I think there are people who are using it, like Povelitel. Is he ok with change?